packages/cardpay-sdk/sdk/reward-manager/base.ts

```diff
@@ -428,6 +429,11 @@ export default class RewardManager {
     let token = new this.layer2Web3.eth.Contract(ERC20ABI as AbiItem[], tokenAddress);
     let safeBalance = new BN(await token.methods.balanceOf(safeAddress).call());
 
+    let rewardSafeDelegateAddress = await getAddress('rewardSafeDelegate', this.layer2Web3);
```

Comment:
I think this is what we just discussed r.e. querying this from the reward manager contract